Slack.Automation/Promact.Erp.Web/Promact.Erp.Web.csproj

```diff
@@ -1000,6 +1004,7 @@
     <Content Include="tslint.json" />
     <Content Include="StringLiteral\StringLiteral.json" />
     <Content Include="tsconfig-aot.json" />
+    <None Include="Properties\PublishProfiles\Promact.ERP.pubxml" />
```

Comment:
I have removed Promact.ERP.pubxml.

but it would be better if you can explain why this file should not be part of the project